src/App.js

```diff
@@ -1,12 +1,26 @@
-import React, { Component } from 'react';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+} from 'react-router-dom';
+
 import './App.css';
+import LoginPage from './views/login';
+import HomePage from './views/home';
+import ErrorNotFound from './views/notfound';
+
 
-class App extends Component {
-  render() {
-    return (
-      <h1>  Welcome to Authors-Haven</h1>
-    );
-  }
-}
+const App = () => (
```

Comment:
@balaakagordon Since we will have to use state, could you please revert this back to a class-based component.